concent_api/core/models.py

```diff
@@ -684,3 +686,38 @@ class PendingEthereumTransaction(Model):
     s = DecimalField(max_digits=78, decimal_places=0)
 
     created_at = DateTimeField(auto_now_add=True)
+
+
+class DepositAccount(Model):
+    client = ForeignKey(Client)
+    ethereum_public_key = CharField(max_length=ETHEREUM_ADDRESS_LENGTH)
```

Comment:
you can add check in `clean` that `ethereum_public_key` is exactly `ETHEREUM_ADDRESS_LENGTH` long, otherwise raise exception

Reply:
Done.

Reply:
Done.